web/packages/test/scripts/build-binary.sh

```diff
@@ -17,7 +17,8 @@ build_binaries() {
     # Check that all 3 binaries are available and no changes made in the polkadot and substrate dirs
     if [[ ! -e "target/release/polkadot" || ! -e "target/release/polkadot-execute-worker" || ! -e "target/release/polkadot-prepare-worker" || "$changes_detected" -eq 1 ]]; then
         echo "Building polkadot binary, due to changes detected in polkadot or substrate, or binaries not found"
-        cargo build --release --locked --bin polkadot --bin polkadot-execute-worker --bin polkadot-prepare-worker
+        # Increase session length to 6 mins
+        ROCOCO_EPOCH_DURATION=60 cargo build --release --locked --bin polkadot --bin polkadot-execute-worker --bin polkadot-prepare-worker
```

Comment:
6 minutes is a long time to wait to test handover cases. I would shorten the session to 20 blocks.

Reply:
https://github.com/Snowfork/snowbridge/pull/1098/commits/95f024c9b4c233e3b6a7d041a45953318efc12ea